app/uk/gov/hmrc/transitmovementsauditing/models/AuditType.scala

```diff
@@ -52,69 +81,80 @@ object AuditType {
-  case object AddMessageDBFailedEvent               extends AuditType("AddMessageDBFailedEvent", commonTransitConventionTraders, None)
-  case object PushNotificationUpdateFailedEvent     extends AuditType("PushNotificationUpdateFailedEvent", commonTransitConventionTraders, None)
-  case object SubmitAttachMessageFailedEvent        extends AuditType("SubmitAttachMessageFailedEvent", commonTransitConventionTraders, None)
-  case object GetMovementsDBFailedEvent             extends AuditType("GetMovementsDBFailedEvent", commonTransitConventionTraders, None)
-  case object GetMovementDBFailedEvent              extends AuditType("GetMovementDBFailedEvent", commonTransitConventionTraders, None)
-  case object GetMovementMessagesDBFailedEvent      extends AuditType("GetMovementMessagesDBFailedEvent", commonTransitConventionTraders, None)
-  case object GetMovementMessageDBFailedEvent       extends AuditType("GetMovementMessageDBFailedEvent", commonTransitConventionTraders, None)
-  case object PushPullNotificationGetBoxFailedEvent extends AuditType("PushPullNotificationGetBoxFailedEvent", commonTransitConventionTraders, None)
+      extends AuditType("InformationAboutNonArrivedMovement", commonTransitConventionTraders, Some(MessageType.IE141), None)
+  case object WriteOffNotification             extends AuditType("WriteOffNotification", transitMovementsRouter, Some(MessageType.IE045), None)
+  case object NoReleaseForTransit              extends AuditType("NoReleaseForTransit", transitMovementsRouter, Some(MessageType.IE051), None)
+  case object RequestOfRelease                 extends AuditType("RequestOfRelease", commonTransitConventionTraders, Some(MessageType.IE054), None)
+  case object GuaranteeNotValid                extends AuditType("GuaranteeNotValid", transitMovementsRouter, Some(MessageType.IE055), None)
+  case object RejectionFromOfficeOfDeparture   extends AuditType("RejectionFromOfficeOfDeparture", transitMovementsRouter, Some(MessageType.IE056), None)
+  case object RejectionFromOfficeOfDestination extends AuditType("RejectionFromOfficeOfDestination", transitMovementsRouter, Some(MessageType.IE057), None)
+  case object ControlDecisionNotification      extends AuditType("ControlDecisionNotification", transitMovementsRouter, Some(MessageType.IE060), None)
+
+  case object ForwardedIncidentNotificationToED
+      extends AuditType("ForwardedIncidentNotificationToED", commonTransitConventionTraders, Some(MessageType.IE182), None)
+  case object PositiveAcknowledge         extends AuditType("PositiveAcknowledge", transitMovementsRouter, Some(MessageType.IE928), None)
+  case object RequestOnNonArrivedMovement extends AuditType("RequestOnNonArrivedMovement", transitMovementsRouter, Some(MessageType.IE140), None)
+
+  case object LargeMessageSubmissionRequested
+      extends AuditType("LargeMessageSubmissionRequested", commonTransitConventionTraders, None, Some("CTCTradersWorkflow"))
```

Comment:
Why are these parent types strings, and worse, strings that are repeated? Please make these enum-like case objects, perhaps `ParentAuditType`, and not just strings.